core/vm/interpreter.go

```diff
@@ -83,14 +83,14 @@ func NewEVMInterpreter(evm *EVM, cfg Config) *EVMInterpreter {
 		}
 		var extraEips []int
 		for _, eip := range cfg.ExtraEips {
```

Comment:
```
		if len(cfg.ExtraEips) > 0 {
			// Deep-copy jumptable to prevent modification of opcodes in other tables
			cfg.JumpTable := copyJumpTable(cfg.JumpTable)
		}
		for _, eip := range cfg.ExtraEips {
			if err := EnableEIP(eip, &cfg.JumpTable); err != nil {
				// Disable it, so caller can check if it's activated or not
				log.Error("EIP activation failed", "eip", eip, "error", err)
			} else {
				extraEips = append(extraEips, eip)
			}
		}
```
Would this work? No need to deep-copy it multiple times

Reply:
Ok, the original one set `cfg.JumpTable` unconditionally, so it don't prevent a partial write neither.

Reply:
I changed to your version